paimon-core/src/main/java/org/apache/paimon/rest/responses/ErrorResponse.java

```diff
@@ -75,10 +71,14 @@ public String getMessage() {
     }
 
     @JsonGetter(FIELD_RESOURCE_TYPE)
-    public ErrorResponseResourceType getResourceType() {
+    public String getResourceType() {
         return resourceType;
     }
 
+    public ErrorResponseResourceType getResourceTypeEnum() {
```

Comment:
This looks weird, can we always use String? We can introduce string constants in this class.